client/src/components/user-actions/user-actions.html

```diff
@@ -0,0 +1,17 @@
+<div class="user-actions">
+    <div data-bind="if: user()">
+        Hello, <span data-bind="text: user().name"></span>
+        |
+        <a href="/#" data-bind="click: logOut">log out</a>
+    </div>
+    <div data-bind="if: !user()">
+        <a href="/#" data-bind="click: () => openForm('sign-in')">Sign in</a>
```

Comment:
Consider moving string literals to view model and creating separate methods like `openSignInForm` and `openSignUpForm`.